search/src/Components/upload/upload_stepper.js

```diff
@@ -241,43 +235,26 @@ class UploadStepper extends React.Component {
 
     handleValidation(formKey, status){
```

Comment:
can't we just get "formKey", i.e. stepName from `this.state.activeStep`? Why does it need to be passed in?